api_server.py

```
@@ -503,10 +553,129 @@ async def chat(
+        elif kind == "error":
+            yield f"data: {json.dumps({'type': 'error', 'message': data})}\n\n"
+            break
+        else:  # done
+            break
+
+    assembled = "".join(full_answer)
+    if needs_translation and assembled:
+        try:
+            translated = await run_in_threadpool(translation.translate_from_english, assembled, language)
+            yield f"data: {json.dumps({'type': 'chunk', 'text': translated})}\n\n"
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)